core/src/subgraph/instance_manager.rs

```diff
@@ -323,7 +325,7 @@ where
             .and_then(|x| x)?;
         }
 
-        let manifest = {
+        let manifest: SubgraphManifest<C> = {
             info!(logger, "Resolve subgraph files using IPFS");
```

Comment:
Not 100% sure if this is desirable, but I had to specify the concrete type here. Otherwise, it would use the `BaseSubgraphManifest` type, which has no methods.

Reply:
This is fine, it's just a type hint.